api/src/controllers/team.js

```diff
@@ -246,6 +246,11 @@ async function deployToGHPages(req, res, next) {
 async function updateTeam(req, res, next) { // eslint-disable-line no-unused-vars
   const { team_id: _id } = req.params;
   const team = req.body;
+  if(team["password"]){
+    const salt = await bcrypt.genSalt();
+    const hashedPassword = await bcrypt.hash(team["password"], salt);
+    team.password = hashedPassword;
+  }
```

Comment:
Better to remove the password property (sensitive info) from the updateTeam before returning it

Reply:
hi, this is by default not returned